citi-onboarding/celeiro-dos-bambas
# Feature(Impact): Criação da seção Impact

## What I did

- Creation of Impact section.
## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   . `git checkout develop`
   . `git pull`
   . `git checkout <feature/impact>`
   . `git pull origin <feature/impact>`
   . `yarn install`
   . `yarn start`

2. In your browser go to http://localhost:3000

3. Check if the component looks just like the mockup in different screen sizes

4. Read the changed files section on GitHub

5. Test the things that you think are worth testing, even the ones that are not in this description
